common/lifecyleMgr.go

```diff
@@ -558,6 +562,19 @@ func (lcm *lifecycleMgr) E2EEnableAwaitAllowOpenFiles(enable bool) {
 	}
 }
 
+func (lcm *lifecycleMgr) SetForceLogging() {
```

Comment:
please add a comment to explain why we fetch this value only once and save it here in lcm, for future reference.

Reply:
Sure. I'll add that.
As for the test run, I did test the functionality by performing a transfer with error and it didn't write anything in the linux syslog. So I guess handling `forcelog` was the key to handle it.

Reply:
It'd be good to do a quick search to see whether we use the default pipeline provided by the SDK.